rollup.config.js

```diff
@@ -9,6 +9,20 @@ import { dependencies } from './package-lock.json'
 
 import fs from 'fs'
 import path from 'path'
+import write from 'write'
+
+import * as edge from 'edge.js'
+edge.registerViews(path.join(__dirname, './packages/shell-chrome/views'))
+const renderTemplate = () => {
+    let template = edge.render('master')
+
+    if (process.env.NODE_ENV === 'production') {
+        template = template.replace(/:data-testid="[^"]*"/g, '')
+    }
+
+    write.sync('./dist/chrome/panel.html', template)
```

Comment:
Any reason we don't want to use `fs.writeFileSync`? Creating the recursive directories I'm guessing?

Reply:
Ha! That's me being not very regular in node 😬 Pretty sure I've used that very method before. Because the template is all rendered at that point we're only writing to one file, so should be fine to use `writeFileSync`, will make that change 👍🏼

Reply:
@HugoDF Completely blasted past what you'd said above...just dawned on me re the directories after the change over to `fs.writeFileSync`. Have added a directory check and recursive `fs.mkdirSync` call to correct.